### ISSUE TITLE
Problem with Pytorch3D

### ISSUE THREAD

**Issue by sIHURs** [PRBonn/LiDiff#15]
Hi!!!,

thank you for the work at first, which is very great!!! I am currently working on a project at Uni based on your content, but I am encountering some difficulties in downloading PyTorch3D on my Windows 10 computer. At this stage, it is quite challenging to borrow a suitable device or rent an appropriate server. I would like to ask if there are any possible alternatives to this dependency for PyTorch3D?

Cheers,
Yifan

**Comment by nuneslu** [comment]
Hi! Thanks for your interest in our work. Regarding PyTorch3d, you can check this previous [issue](https://github.com/PRBonn/LiDiff/issues/9). For PyTorch3d, the only function used is the `chamfer_distance`, which is used only during the refinement network training to compute the chamfer distance loss. So, you don't need it to train the diffusion part of the model.

For the refinement part, you could look for other options for computing the chamfer distance loss, e.g., this [package](https://pypi.org/project/chamferdist/). You should be able to install it using pip, but be aware that since it is a different implementation of the chamfer distance, this may lead to different results than the ones reported in the paper.

**Comment by nuneslu** [comment]
I will close the issue now since there was no update. But if you still have a problem with it, feel free to reopen it.
